{"filter":false,"title":"simple_topic_publisher.py","tooltip":"/my_package/src/simple_topic_publisher.py","undoManager":{"mark":0,"position":0,"stack":[[{"start":{"row":0,"column":0},"end":{"row":14,"column":14},"action":"insert","lines":["#! /usr/bin/env python","","import rospy","from std_msgs.msg import Int32 ","","rospy.init_node('topic_publisher')","pub = rospy.Publisher('/counter', Int32, queue_size=1)","rate = rospy.Rate(2)","count = Int32()","count.data = 0","","while not rospy.is_shutdown(): ","  pub.publish(count)","  count.data += 1","  rate.sleep()"],"id":1}]]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":10,"column":0},"end":{"row":10,"column":0},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1547097575216,"hash":"ace5d3c1233504d3fb4bacfa096c85ef9635531a"}